frontend/asset_mapper.rst

```diff
@@ -93,6 +93,9 @@ This will physically copy all the files from your mapped directories to
 ``public/assets/`` so that they're served directly by your web server.
 See :ref:`Deployment <asset-mapper-deployment>` for more details.
 
+If you compiled your assets on your development machine, you need to delete them again,
```

Comment:
`development machine` or `dev environment` ?

also I would at it in an info block, to make things more visible, wdyt?

Reply:
OK, done :-)